dashboard/modules/dashboard_sdk.py

```diff
@@ -135,25 +136,24 @@ def parse_cluster_info(
                 "http://"
                 f"{ray._private.worker.global_worker.node.address_info['webui_url']}"
             )
+            logger.info(
+                f"No address provided but Ray is running; using address {address}."
+            )
         else:
             logger.info(
                 f"No address provided, defaulting to {DEFAULT_DASHBOARD_ADDRESS}."
             )
             address = DEFAULT_DASHBOARD_ADDRESS
 
-    module_string, inner_address = _split_address(address)
-
-    # If user passes in ray://, raise error. Dashboard submission should
-    # not use a Ray client address.
-    if module_string == "ray":
-        raise ValueError(
-            f'Got an unexpected Ray client address "{address}" while trying '
-            "to connect to the Ray dashboard. The dashboard SDK requires the "
-            "Ray dashboard server's HTTP(S) address (which should start with "
-            '"http://" or "https://", not "ray://"). If this address '
-            "wasn't passed explicitly, it may be set in the RAY_ADDRESS "
-            "environment variable."
+    if "://" not in address:
+        # Default to HTTP.
+        logger.info(
```

Comment:
should this be warning? we do want users to be expressive here? also how are we recommending users in documentation right now?

Reply:
That's fair, I'll change it to a warning.  Right now in the State CLI docs, there's nothing written about the address besides "Ray bootstrap address, could be auto, localhost:6379. If None, it will be resolved automatically from an initialized ray."  But `localhost:6739` gets internally converted to the dashboard api URL, with an `http://` in front of it.

In the Ray Jobs docs, currently we say to use `http://<host>:8265`, but I'll update the docs to include the option to use `ray://` after this PR.